Changing ownership of dump/coco fails during vagrant up

I'm getting the following error when I'm running ```vagrant up```. Keep in mind I'm running Vagrant with VMware Fusion instead of Virtualbox. I wouldn't think that would cause this particular problem, but I thought I would mention it.

I'm running Mac OS X, v. 10.10.1. Because of that, there's no such thing as an "administrative command prompt". I've tried to run ```git clone``` and ```vagrant up``` with the ```sudo``` command, but that causes other problems.

I tried to simply ignore the problem and run ```vagrant ssh -c "cd /coco && brunch watch"```, but I get another error from node.js saying that it can't find lodash.

```
==> default: dump/
==> default: dump/coco/
==> default: dump/coco/levels.bson
==> default: tar:
==> default: dump/coco/levels.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/campaigns.metadata.json
==> default: tar:
==> default: dump/coco/campaigns.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/campaigns.bson
==> default: tar:
==> default: dump/coco/campaigns.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/patches.bson
==> default: tar:
==> default: dump/coco/patches.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/articles.bson
==> default: tar:
==> default: dump/coco/articles.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/patches.metadata.json
==> default: tar:
==> default: dump/coco/patches.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/level.systems.bson
==> default: tar:
==> default: dump/coco/level.systems.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/achievements.metadata.json
==> default: tar:
==> default: dump/coco/achievements.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/level.components.bson
==> default: tar:
==> default: dump/coco/level.components.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/thang.types.bson
==> default: tar:
==> default: dump/coco/thang.types.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/media.chunks.metadata.json
==> default: tar:
==> default: dump/coco/media.chunks.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/media.files.metadata.json
==> default: tar:
==> default: dump/coco/media.files.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/achievements.bson
==> default: tar:
==> default: dump/coco/achievements.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/thang.types.metadata.json
==> default: tar:
==> default: dump/coco/thang.types.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/media.files.bson
==> default: tar:
==> default: dump/coco/media.files.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/level.components.metadata.json
==> default: tar:
==> default: dump/coco/level.components.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/articles.metadata.json
==> default: tar:
==> default: dump/coco/articles.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/levels.metadata.json
==> default: tar:
==> default: dump/coco/levels.metadata.json: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/media.chunks.bson
==> default: tar:
==> default: dump/coco/media.chunks.bson: Cannot change ownership to uid 1000, gid 1000
==> default: : Operation not permitted
==> default: dump/coco/level.systems.metadata.json
==> default: tar: dump/coco/level.systems.metadata.json: Cannot change ownership to uid 1000, gid 1000: Operation not permitted
==> default: tar: dump/coco: Cannot change ownership to uid 1000, gid 1000: Operation not permitted
==> default: tar: dump: Cannot change ownership to uid 1000, gid 1000: Operation not permitted
==> default: tar: Exiting with failure status due to previous errors
```

I would really love to get this working and merged into codecombat's code via [#2106](https://github.com/codecombat/codecombat/issues/2106), so please let me know if I can provide further information or support!

Fantastic, that part works now. Thanks!

Unfortunately, I'm getting the following error when running ```vagrant ssh -c "cd /coco && brunch watch"```:

```
Error: couldn't load config /coco/config.coffee. Error: Cannot find module 'lodash'
  at Object.exports.loadConfig (/usr/lib/node_modules/brunch/lib/helpers.js:514:15)
  at Object.exports.loadConfig (/usr/lib/node_modules/brunch/lib/helpers.js:512:24)
  at initialize (/usr/lib/node_modules/brunch/lib/watch.js:453:20)
  at new BrunchWatcher (/usr/lib/node_modules/brunch/lib/watch.js:607:7)
  at module.exports.watch (/usr/lib/node_modules/brunch/lib/watch.js:663:12)
  at start (/usr/lib/node_modules/brunch/lib/index.js:13:33)
  at Command.<anonymous> (/usr/lib/node_modules/brunch/node_modules/commander/index.js:249:8)
  at Command.emit (events.js:98:17)
  at Command.parseArgs (/usr/lib/node_modules/brunch/node_modules/commander/index.js:472:12)
  at Command.parse (/usr/lib/node_modules/brunch/node_modules/commander/index.js:370:21)
  at Object.exports.run (/usr/lib/node_modules/brunch/lib/cli.js:49:13)
  at Object.<anonymous> (/usr/lib/node_modules/brunch/bin/brunch:11:22)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Function.Module.runMain (module.js:497:10)
  at startup (node.js:119:16)
  at node.js:906:3
```

Any ideas on this one? I'll look at it a bit more later.

--- a/fillMongo.sh
+++ b/fillMongo.sh
@@ -2,5 +2,5 @@
 rm -f dump.tar.gz
 rm -rf dump
 wget http://54.91.159.37/dump.tar.gz
-tar xzvf dump.tar.gz
+tar xzvf dump.tar.gz --no-same-owner
 mongorestore

